source/blog/2018-08-03-rspec-3-8-has-been-released.md

```diff
@@ -0,0 +1,112 @@
+This feature is useful when your test suite has an ordering
+dependency--that is, the suite only fails when the tests are run
+in a specific order.  `--bisect` will repeatedly run smaller and
+smaller subsets of your suite in order to narrow it down to the
+minimal set of specs needed to reproduce the failures.
+
+Since 3.3, this feature has been implemented by shelling out to
+the `rspec` command to run each subset. While simple and effective,
+we realized this approach was also quite inefficient. Each time the
+`rspec` command runs, it must boot RSpec and your application
+environment (which may include Rails and many other gems) from scratch.
+The cost of this can vary considerably from a couple hundred milliseconds
+to 30+ seconds on a large Rails app. In extreme cases, the runtime of
+`--bisect` can be dominated by the time it takes to boot your application
+environment over and over and over.
+
+In RSpec 3.8, we've improved bisect's performance by using forking
+on platforms that support it rather than shelling out. That way, we
+can boot your application environment _once_, and then fork a subprocess
+in which to run each subset of the test suite, avoiding the need to boot
+your application many times.
+
+The actual improvement you'll see in practice will vary widely, but
+in our [limited testing](https://github.com/rspec/rspec-core/pull/2511)
+it improved the runtime of `--bisect` by 33% in one case and an
+order-of-magnitude (108.9 seconds down to 11.7 seconds) in another.
+
+If you're looking to maximize the benefit of this change, you may
+want to pass some additional `--require` options when running a
+bisection in order to pre-load as much of your application environment
+as possible.
+
+### Core: Support the XDG base directory spec for configuration
+
+RSpec, like many command line tools, supports the use of options
+files, which can live at `.rspec` (for team project options)
+`~/.rspec` (for global personal options) or at `.rspec-local`
+(for personal project options -- this file should not be under
+source control). In RSpec 3.8, we've expanded this feature to
+support the [XDG Base Directory
+Specification](https://specifications.freedesktop.org/basedir-spec/latest/),
+which defines a standard way for tools to locate the global personal
+options file.  This gives users complete control over where this
+file is located rather than forcing it to live in their home directory.
+
+To use this feature, simply set the `$XDG_CONFIG_HOME` environment
+variable and put your RSpec options at `$XDG_CONFIG_HOME/rspec/options`.
+
+For more info, [read the spec](https://specifications.freedesktop.org/basedir-spec/latest/)
+or [check out the pull
+request](https://github.com/rspec/rspec-core/pull/2538).
+
+Thanks to Magnus Bergmark for implementing this feature!
+
+### Expectations: Formatted output length is now configurable
+
+TODO (Sam/Benoit): write this up since you authored this feature
```

Comment:
How about:

> When dealing with larger objects their string representations can become rather unwieldy and can clutter the console output. In RSpec 3.6 we started truncating these objects to prevent this issue but due to an oversight this was never made easily configurable. In 3.7 RSpec Expectations has a new configuration option allowing changing this value and also allows turning it off (by setting the limit to `nil`).

Reply:
Thanks, I've adapted this.